doc/architecture/20170824-audit-plan-to-model-audit-data.md

```diff
@@ -0,0 +1,36 @@
+## Problem 
+
+1. Discrepancies in the expected numbers of content items that need to be audited when we compare them with the existing spreadsheets. 
+2. Themes do not represent the input of an Audit.
+  
+## Decision
+
+1. Introduce the notion of an `Audit Plan` (defined below) that models the input of an audit.
+2. Do not use Themes in the Auditing tool, and rely on primary organisations.
```

Comment:
~~Auditing tool~~ Content Audit Tool

Reply:
Addressed in d509fe718b81d8e902dd7a0f71f8f00df099a49a